### ISSUE TITLE
Unable to solve dependencies on latest Ubuntu

### ISSUE THREAD

**Issue by Utopiah** [hahnec/plenopticam#23]
Hi, I'm trying to setup in order to test Lytro content on the LookingGlass Portrait.
I tried `python3 -m pip install plenopticam` but ended up in dependency hell. After a bit of search I decided to use pyenv to rollback from Python 3.9 to 3.8 as apparently scipy had problem with it (no package yet). That ended up solving this part but then few more dependencies were still missing, namely sciki-image, which was fine, but also tk-tkinter which I didn't manage to solve : 

```
  File "/home/fabien/.pyenv/versions/3.8.12/lib/python3.8/tkinter/__init__.py", line 36, in <module>
    import _tkinter # If this fails your Python may not be configured for Tk
```

As it gets a bit hairy trying to mix pyenv and system wide setups e.g `sudo apt install python3-tk tk-dev` I'm wondering if a dockerfile wouldn't simplify. What I did for now was :

```
docker run -it --rm  scipy/scipy-dev
python3.8 -m pip install scikit-image
python3.8 -m pip install plenopticam
plenopticam -h
```

which worked.

**Comment by Utopiah** [comment]
In case anybody unfamiliar with Docker/podman needs this :

Dockerfile : 
```
FROM scipy/scipy-dev
RUN python3.8 -m pip install scikit-image
RUN python3.8 -m pip install plenopticam
RUN plenopticam -h
```

then build it with `docker build . -t pleno` and start with `docker run -it --rm pleno`. If you need to share a local directory check the documentation on volumes.

Note of caution, this is headless so `-g` will not work.

**Comment by hahnec** [comment]
Hi Fabien, Thanks for reporting this issue and pointing to a nice workaround. I am currently tracing the roots of this and will post updates here.

**Comment by hahnec** [comment]
I just released [v0.8.2](https://github.com/hahnec/plenopticam/releases/tag/v0.8.2) , which contains the above fixes that worked on  Ubuntu Focal. To test it, you may run

`python3 -m pip install plenopticam==0.8.2`

BTW, the `scikit-image` dependency issue was not limited to Ubuntu, but also failed on other platforms. So, thanks again for pointing this out :+1: Please re-open this case if [v0.8.2](https://github.com/hahnec/plenopticam/releases/tag/v0.8.2)  continues to raise such problems on your environment.

**Comment by Utopiah** [comment]
Very nice, `python3 -m pip install plenopticam==0.8.2` worked instantly. No need to downgrade to 3.8 nor use Docker. Thanks for such prompt fix and for your research!
